inscriptions/ISic000696.xml

```diff
@@ -191,14 +196,21 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#JH">
+                <ab>
+                    <lb n="1"/><name>Iulia</name> <name>Gamice</name>
```

Comment:
I'm not at all clear what is going on here, but you can't put the text of a different inscription into this file - the file has all its background information including bibliography in it already, and the Isicily number is a fixed and non-changeable identifier - you *MUST* encode the ISic696 inscription in this file, not another one. As it says in the bibliography, this inscripiton is Manganaro 1989, p.165 no.25 fig.25 (not p.195 no.91!). In this case, I would additionally recommend that you encode the text as re-edited by Buoncore in the amphitheatre epigraphy corpus, image of the text pasted here:
![image](https://user-images.githubusercontent.com/25080386/121251195-8499ec00-c89e-11eb-9648-a68c85dea8e9.png)